NSRetailPOS/NSRetailPOS/NSRetailPOS/PaymentGatewayHelper.cs

```diff
@@ -0,0 +1,147 @@
+﻿using DevExpress.Office.Utils;
+using DevExpress.XtraEditors;
+using DevExpress.XtraRichEdit.Import.Html;
+using DevExpress.XtraSplashScreen;
+using Newtonsoft.Json;
+using NSRetailPOS.UI;
+using System;
+using System.Collections.Generic;
+using System.Data;
+using System.Net.Http;
+using System.Text;
+using System.Threading.Tasks;
+using System.Windows.Forms;
+
+namespace NSRetailPOS
+{
+    public enum PaymentMode
+    {
+        Card = 1,
+        UPI = 10
+    }
+    public class PaymentGatewayHelper
+    {
+        public static async Task TestPaymentAsync(PaymentMode paymentMode,string paymentUrl,int sequenceNumber = 1, int amountInSmallestUnit = 1)
+        {
+            var payload = new RequestPayload
+            {
+                TransactionNumber = GenerateTransactionNumber(),
```

Comment:
Is this transaction number our BillNumber? Can we add /1 /2 in the end for multiple cards? so basically transaction number will be counter/date/prefix0s/billnumber/1... even for one card lets put /1 always so that we don't need extra logic while retracing?